docs/Roadmap.md

```diff
@@ -0,0 +1,18 @@
+# Roadmap
+
+## Q2'24
```

Comment:
version names? `v0.4.0`, `v0.5.0`, etc.

Reply:
its too early to know assign it version numbers.